front/src/applications/operationalStudies/views/ManageTrainSchedule.tsx

```diff
@@ -152,14 +152,18 @@ const ManageTrainSchedule = () => {
         <div className="floating-itinerary">
           <Itinerary rollingStockId={rollingStockId} />
         </div>
-
-        <Map
-          pathProperties={pathProperties}
-          simulationPathSteps={markersInformation}
-          pathStepsAndSuggestedOPs={pathStepsAndSuggestedOPs}
-        >
-          <IncompatibleConstraints pathProperties={pathProperties} />
-        </Map>
+        {pathProperties && (
+          <Map
+            pathProperties={{
+              length: pathProperties?.length,
+              geometry: pathProperties?.geometry,
```

Comment:
`pathProperties` is already guaranteed to be defined in this section, so the non-null assertion (`!`) can be removed.
```suggestion
              length: pathProperties.length,
              geometry: pathProperties.geometry,
```

Reply:
fixed !